### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by SiegfriedBz**

### CODE REVIEW

File: app/token/[address]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// import type { TToken } from "../../../types";
+
+type TProps = {
```
Comment (dcts) [inline app/token/[address]/page.tsx:3]: what does TProps stand for?

File: app/launch/page.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const page = () => {
```
Comment (dcts) [inline app/launch/page.tsx:1]: Is this standard next convention? Should it not be 
```
const Launch = () => {}

export default Launch;
```

I'm happy with whatever was just wondering.

File: types/index.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export type TToken = {
+  name: string;
+  symbol: string;
+  image: string;
+  description: string;
+  telegramLink?: string;
```
Comment (dcts) [inline types/index.ts:6]: `telegram` should be sufficient.

File: types/index.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  xLink?: string;
```
Comment (dcts) [inline types/index.ts:7]: `twitter` maybe?

**Review by dcts — APPROVED** [review]

File: app/token/[address]/page.tsx (hunk 1 above)
Comment (SiegfriedBz) [reply app/token/[address]/page.tsx:3]: Type of Props ;)

File: app/launch/page.tsx (hunk 2 above)
Comment (SiegfriedBz) [reply app/launch/page.tsx:1]: oh, it s just a vscode snippet I m using, to generate a React component. it takes the name of the file to generate the name of the component

File: types/index.ts (hunk 4 above)
Comment (SiegfriedBz) [reply types/index.ts]: xLink, I just changed it to xUrl

**Pull request merged**